test/e2e/flowaggregator_test.go

```diff
@@ -188,16 +210,53 @@ func TestFlowAggregator(t *testing.T) {
 	}
 
 	if v4Enabled {
-		t.Run("IPv4", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, false) })
+		t.Run("IPv4", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, false, false) })
 	}
 
 	if v6Enabled {
-		t.Run("IPv6", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, true) })
+		t.Run("IPv6", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, true, false) })
+	}
+
+}
+
+func secureConnectionHelper(t *testing.T, o tlsTestOptions) {
```

Comment:
it doesn't really make sense to define test cases within a function called `*Helper`. However, it looks it was already done in that file.

Reply:
I was thinking to reuse the code in that file. I've updated it. Thanks.